From cb4f552d230f61e142c40606a06e8ef813a435b7 Mon Sep 17 00:00:00 2001
From: ned <ned@need>
Date: Tue, 22 Apr 2025 23:13:00 +0000
Subject: [PATCH] fix patchs fails trebledroid

---
 init/first_stage_mount.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/first_stage_mount.cpp b/init/first_stage_mount.cpp
index 2c1fdad..b2ac402 100644
--- a/init/first_stage_mount.cpp
+++ b/init/first_stage_mount.cpp
@@ -468,7 +468,7 @@ bool FirstStageMountVBootV2::MountPartition(const Fstab::iterator& begin, bool e
     if (begin->fs_mgr_flags.avb) {
         if (!SetUpDmVerity(&(*begin))) {
             PLOG(ERROR) << "Failed to setup verity for '" << begin->mount_point << "'";
-            return false;
+            //return false;
         }
     } else {
         LOG(INFO) << "AVB is not enabled, skip verity setup for '" << begin->mount_point << "'";
-- 
2.43.0

