From c994834aa4aa27225567c2b1c0ac843ab6faf9f3 Mon Sep 17 00:00:00 2001
From: Jotanh <joaopedromelen2910@gmail.com>
Date: Tue, 29 Apr 2025 03:23:11 +0000
Subject: [PATCH 2/2] init-Do-not-start-console-service-when-debuggable

---
 rootdir/init.rc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 3096cbe6b..c790de17b 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1392,8 +1392,8 @@ on property:ro.debuggable=1
     # Give reads to anyone for the accessibility trace folder on debug builds.
     chmod 0775 /data/misc/a11ytrace
 
-on init && property:ro.debuggable=1
-    start console
+
+
 
 # Allow other processes to run `snapshotctl` through `init`. This requires
 # `set_prop` permission on `snapshotctl_prop`.
-- 
2.43.0

