From 18546abab0d2e519989a56e007890db6e6d9c64f Mon Sep 17 00:00:00 2001
From: ned <ned@need>
Date: Tue, 22 Apr 2025 17:41:08 +0000
Subject: [PATCH] controller.cc os::

---
 system/gd/hci/controller.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/system/gd/hci/controller.cc b/system/gd/hci/controller.cc
index 0605dae..8b383f7 100644
--- a/system/gd/hci/controller.cc
+++ b/system/gd/hci/controller.cc
@@ -647,7 +647,7 @@ struct Controller::impl {
     vendor_capabilities_.is_supported_ = 0x01;
 
     int vendor_cap_max = 0xffff;
-    std::string vendor_cap_max_prop = GetSystemProperty("persist.sys.bt.max_vendor_cap").value_or("");
+    std::string vendor_cap_max_prop = os::GetSystemProperty("persist.sys.bt.max_vendor_cap").value_or("");
     if (vendor_cap_max_prop != "") {
         vendor_cap_max = std::stoi(vendor_cap_max_prop);
     }
-- 
2.43.0

