From a4c9c3c65e0132c610626c57e8d37e6b9cf52f63 Mon Sep 17 00:00:00 2001
From: ned <ned@need>
Date: Tue, 22 Apr 2025 17:19:22 +0000
Subject: [PATCH 1/2] remove duplicade ro.device

---
 config/version.mk | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/config/version.mk b/config/version.mk
index 330a595..fed551e 100644
--- a/config/version.mk
+++ b/config/version.mk
@@ -16,7 +16,7 @@ ANDROID_VERSION := 15
 INFINITYVERSION := 2.7
 
 INFINITY_BUILD_TYPE ?= UNOFFICIAL
-INFINITY_MAINTAINER ?= UNKNOWN
+INFINITY_MAINTAINER ?= PAPACU
 INFINITY_DATE_YEAR := $(shell date -u +%Y)
 INFINITY_DATE_MONTH := $(shell date -u +%m)
 INFINITY_DATE_DAY := $(shell date -u +%d)
@@ -56,6 +56,5 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
   ro.infinity.build.date=$(INFINITY_BUILD_DATE) \
   ro.infinity.buildtype=$(INFINITY_BUILD_TYPE) \
   ro.infinity.fingerprint=$(INFINITY_FINGERPRINT) \
-  ro.infinity.device=$(INFINITY_BUILD) \
   ro.infinity.version=$(INFINITYVERSION) \
   ro.infinity.maintainer=$(INFINITY_MAINTAINER)
-- 
2.43.0

