From 5120aab377ce058680e0b04ef3522b9033e2315f Mon Sep 17 00:00:00 2001
From: ned <ned@need>
Date: Tue, 22 Apr 2025 17:33:50 +0000
Subject: [PATCH 2/2] remove duplicade ro.adb

---
 config/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/common.mk b/config/common.mk
index 39a5442..0128c3c 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -27,7 +27,7 @@ ifeq ($(TARGET_BUILD_VARIANT),eng)
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
 else
 # Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
+#PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
 
 # Disable extra StrictMode features on all non-engineering builds
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.strictmode.disable=true
-- 
2.43.0

