From ffa79291ff8e5854a80a9418e6fa87ca491df89e Mon Sep 17 00:00:00 2001
From: Jotanh <joaopedromelen2910@gmail.com>
Date: Sun, 27 Apr 2025 18:44:32 +0000
Subject: [PATCH 3/3] remove su

---
 target/product/base_system.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/target/product/base_system.mk b/target/product/base_system.mk
index 5ce6613..35379f9 100644
--- a/target/product/base_system.mk
+++ b/target/product/base_system.mk
@@ -527,7 +527,6 @@ PRODUCT_PACKAGES_DEBUG := \
     ss \
     start_with_lockagent \
     strace \
-    su \
     sanitizer-status \
     tracepath \
     tracepath6 \
-- 
2.43.0

